.github/workflows/feature-list.yml

```diff
@@ -37,39 +29,72 @@ jobs:
         run: |
           npm install csvtojson --legacy-peer-deps
 
-      - name: Fetch spreadsheet and check for updates
+      - name: Fetch spreadsheet and process updates
         run: |
           # Download the spreadsheet
           curl -L $SPREADSHEET_URL -o spreadsheet.csv
-          
-          # Generate a SHA hash of the file
-          sha256sum spreadsheet.csv > .sha-cache/latest-sha
-          # Compare the hash with the previously stored hash
-          if [ -f ".sha-cache/last-sha" ] && cmp --silent .sha-cache/latest-sha .sha-cache/last-sha; then
-            echo "No updates detected."
-            echo "has-updates=false" >> $GITHUB_OUTPUT
-          else
-            echo "Updates detected. Processing..."
-            cp .sha-cache/latest-sha .sha-cache/last-sha
-            # Convert CSV to JSON (Assume a script or tool to do this is present)
-            node -e "
-              const fs = require('fs');
-              const csv = require('csvtojson');
-              csv()
-                .fromFile('spreadsheet.csv')
-                .then(json => fs.writeFileSync(process.env.FEATURES_FILE, JSON.stringify(json, null, 2)));
-            "
-            echo "has-updates=true" >> $GITHUB_OUTPUT
-          fi
+
+          # Process the CSV, filter data, and append to feature_data.json
+          node -e "
+          const fs = require('fs');
+          const csv = require('csvtojson');
+
+          // Read and parse CSV
+          csv()
+            .fromFile('spreadsheet.csv')
+            .then(rows => {
+              const filteredData = rows.map(row => {
+                const includeRow = 
+                  (row['pricing page?'] && ['x', 'X', 'true'].includes(row['pricing page?'].toLowerCase())) || 
+                  (row['documented?'] && (row['documented?'].startsWith('https://docs.meshery.io/') || row['documented?'].startsWith('https://docs.layer5.io/')));
```

Comment:
Why do we care about this here?

Reply:
I'll remove documented from this one.

Reply:
We'll need the documented rows to add a hyperlink to the documentation pages @leecalcote